eclipse-collections-code-generator/src/main/resources/test/factory/primitiveObjectMapsTest.stg

```diff
@@ -0,0 +1,69 @@
+import "copyright.stg"
+import "primitiveLiteral.stg"
+
+isTest() ::= "true"
+
+targetPath() ::= "org/eclipse/collections/impl/map/primitive"
+
+fileName(primitive) ::= "<primitive.name>ObjectMapsTest"
+
+skipBoolean() ::= "true"
+
+class(primitive) ::= <<
+<body(primitive.type, primitive.name, primitive.wrapperName, primitive.charPrimitive)>
+>>
+
+body(type, name, wrapperName, charPrimitive) ::= <<
+<copyright()>
+
+package org.eclipse.collections.impl.map.primitive;
```

Comment:
Package should be `factory` instead of `map` since we're testing the factories.

Reply:
agreed